03_letters/brvishnu92/main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+	expected := `a:2
+b:1
+`
+	actual := buf.String()
+	assert.Equalf(t, expected, actual, "got %v want %v", actual, expected)
+}
+
+func TestLetters(t *testing.T) {
+	testCases := map[string]struct {
+		input    string
+		expected map[rune]int
+	}{
+		"test1":   {input: "blahbla", expected: map[rune]int{97: 2, 98: 2, 104: 1, 108: 2}},
```

Comment:
You should give your tests meaningful names, and only add tests that test additional cases. For "test1" and "test2", they are really testing the same thing, so you only need one of them. Additional unnecessary tests just add a maintenance burden for no additional gain.

If you can come up with meaningful names for your tests, you can often see that two tests would have the same name, so there's no point having both.